controllers/accounts.go

```diff
@@ -3,21 +3,122 @@ package controllers
 import (
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"net/http"
 
 	ledgerContext "github.com/RealImage/QLedger/context"
 	"github.com/RealImage/QLedger/models"
 )
 
-func GetAccountInfo(w http.ResponseWriter, r *http.Request, context *ledgerContext.AppContext) {
-	accountsDB := models.AccountDB{DB: context.DB}
+func GetAccounts(w http.ResponseWriter, r *http.Request, context *ledgerContext.AppContext) {
+	engine, err := models.NewSearchEngine(context.DB, "accounts")
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	defer r.Body.Close()
```

Comment:
Can we move this before initialising NewSearchEngine? If it returns error we would not really use the engine object.